css/styles.css

```diff
@@ -0,0 +1,174 @@
+* { 
+  box-sizing: border-box; 
+}
+
+body {
+  background-color: none;
+  width: 960px;
+  height: 100vh;
+  font-family: "Source Sans Pro";
+  margin: 0px;
+}
+
+h1 {
+  color:yellow;
+  position: relative;
+  left: 3%;
+  top: 13%;
+}
+
+nav {
+  position: relative;
+  right: 45px;
+  top:5px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-content: space-between;
+}
+
+nav a {
+  margin-left: 15px;
+}
+
+span {
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+u {
+  text-decoration-color: yellow;
+}
+
+a {
+  color:white;
+  text-decoration: none;
+}
+
+h3 {
+  font-weight: bolder;
+  font-size: 24px;
+}
+
+h6 {
+  color: white;
+  font-size: 18px;
+  position: relative;
+  left: 5%;
+  top: 35%;
+  margin: 10px, 10px, 10px, 10px;
```

Comment:
@zdburnett Since all the side have the same value this could be `margin: 10px`

Reply:
done!